src/online2bin/online2-wav-nnet2-latgen-faster.cc

```diff
@@ -124,7 +124,7 @@ int main(int argc, char *argv[]) {
                 "--use-most-recent-ivector=true and --greedy-ivector-extractor=true "
                 "in the file given to --ivector-extraction-config, and "
                 "--chunk-length=-1.");
-    po.Register("num-threads-startup", &g_num_threads,
+    po.Register("num-threads", &g_num_threads,
```

Comment:
I don't understand why you want to change this option name.
The reason it was called --num-threads-startup was to avoid giving users the impression that this binary is actually able to do multi-threaded decoding.   (The num-threads is only used at the very beginning of the program).

Reply:
Sure. How about `num-threads-ivectors`? The point is that it uses multithreading even if the user is not aware of it (and it has nothing to do with startup)

Reply:
The ivector extractor [always starts multithreaded](https://github.com/kaldi-asr/kaldi/blob/831dd38cf7f94dff0fe8325d11e27e1eb8946389/src/ivector/ivector-extractor.cc#L198) using `g_num_threads`, ie. if someone uses the ivector extractor, they should be setting `g_num_threads` (which is currently done by the obscure `num-threads-startup` option).